staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -1036,12 +1036,11 @@ func (e *Store) DeleteCollection(ctx context.Context, deleteValidation rest.Vali
 				accessor, err := meta.Accessor(items[index])
 				if err != nil {
 					errs <- err
```

Comment:
by not exiting early, this means a single worker can enqueue multiple errors, and block on a full errs channel